.github/workflows/crc.yaml

```diff
@@ -33,8 +33,15 @@ jobs:
         go-version: ${{ matrix.go }}
     - name: Download and install CRC
       run: |
-        wget "https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/2.40.0/crc-linux-amd64.tar.xz"
-        tar -xf crc-linux-amd64.tar.xz --directory /usr/local/bin --strip-components=1 crc-linux-2.40.0-amd64/crc
+        wget "https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/crc/latest/crc-linux-amd64.tar.xz"
+
+        latest_version=$(git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' https://github.com/crc-org/crc \
+        | tail -n1 \
+        | sed 's/.*\///; s/\^{}//; s/^v//')
+
+        # Use the extracted version in the tar command
+        tar -xf crc-linux-amd64.tar.xz --directory /usr/local/bin --strip-components=1 "crc-linux-${latest_version}-amd64/crc"
```

Comment:
We are doing that because even though it is `latest`, the binary has the version number in the tarball ?

Reply:
As we are building tekton-caches from konflux, should we invest on GH workflows ?

Reply:
> We are doing that because even though it is latest, the binary has the version number in the tarball ?

@vdemeester  Yes, the binary has the version inside the tarball

Reply:
> As we are building tekton-caches from konflux, should we invest on GH workflows ?

@pramodbindal I'm sorry I did not get the context completely

Reply:
@PuneetPunamiya  / @vdemeester 
We are building all  compoennts from konflux including tekton-caches.

so any kind of build/test/release related enhancements should be done there only.

Reply:
@pramodbindal not necessarily no. We use whatever works and depending on the constraints. 
For example, we use konflux to be able to easily release it as part of the Red Hat OpenShift Pipelines operator — if we didn't integrate it with, we wouldn't probably use konflux here and only rely on GitHub workflows.

What we need to make sure is that it is relatively easily to switch from one CI to the other. For example, running e2e tests on crc or on another openshift instance should be a matter of a set of different arguments we pass to a script.

But in general, there is no reason to stick to only on CI system and do everything in it. Let's use what works best for what me need, and we can mix and match.